public/app/features/canvas/runtime/scene.tsx

```diff
@@ -356,6 +359,10 @@ export class Scene {
     return targetElements;
   };
 
+  connectionsNeedUpdate = (element: ElementState): boolean => {
```

Comment:
Could this potentially live in the `connections` class?

Reply:
yes! moved it!